### DISCUSSION TITLE
Optimization with constraints on geo level

### DISCUSSION THREAD

**Discussion by xijianlim** [google/lightweight_mmm#41] [Ideas]
Hi, this was actually been requested by my colleagues.

Is there a way to create a separate optimization on the geo-levels but with the same type of constraints that you have on the national level?

**Comment by pabloduque0** [comment]
Hello @xijianlim  at the moment we only allow for optimization at national level, although we could consider it at geo level as well. It would require a couple extra things but it is definitely doable. Computationally it also gets a bit more intensive.

A question for what the API would look like in what you are looking for:
- Would you want to pass prices per channel only or channel and geo? Because depending on this it could generate some issues in certain cases.

Let me know
